framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1255,6 +1261,15 @@ protected void addComputedTypeAnnotations(Element elt, AnnotatedTypeMirror type)
     }
 
     /**
+     * Adds default annotations to {@code type}.
```

Comment:
We need this method because at the call sites, there is no tree or element that corresponds to `type`.  I did not name this method `addComputedTypeAnnotations` because it only adds `defaults`. 

I added a comment to caution the use of this method.